core/doc/snippets/uServices-resources/main.cpp

```diff
@@ -38,11 +37,11 @@ void parseComponentDefinition(std::istream&)
 {
 }
 
-void extenderPattern()
+void extenderPattern(ModuleContext* moduleCtx)
 {
   //! [2]
   // Get all loaded modules
```

Comment:
Yes, nomenclature changes will correct such occurrences.

FYI; "loaded" would equate to "started". "Install" and "Uninstall" are new states introduced by these changes.